src/openepd/model/specs/generated/finishes.py

```diff
@@ -495,6 +497,10 @@ class PlasterV1(BaseOpenEpdHierarchicalSpec):
 
     _EXT_VERSION = "1.0"
 
+    # Own fields:
+    composition: PlasterComposition | None = pyd.Field(default=None, description="", example="Cement")
```

Comment:
Should `example` be string or enum?

Reply:
Added description for application_rate. Composition does not have description

Reply:
Examples better be strings. When running in unit tests, there was some issue with comparing them against server response (unit-tests were originally generated by the codegen)